app/src/main/kotlin/edu/artic/CrashlyticsTree.kt

```diff
@@ -0,0 +1,39 @@
+package edu.artic
+
+import android.util.Log
+import com.crashlytics.android.Crashlytics
+import timber.log.Timber
+
+
+/**
+ * Sends the timber logs to Crashlytics.
+ * @author Sameer Dhakal (Fuzz)
+ */
+
+class CrashlyticsTree : Timber.Tree() {
+
+    private val reportableLogs: Set<Int> = setOf(Log.ERROR)
+
+    override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {
+
+        if (reportableLogs.contains(priority)) {
+            return
+        }
+
+        Crashlytics.setInt(CRASHLYTICS_KEY_PRIORITY, priority)
+        Crashlytics.setString(CRASHLYTICS_KEY_TAG, tag)
+        Crashlytics.setString(CRASHLYTICS_KEY_MESSAGE, message)
```

Comment:
Please just use the three-args `::log` method.

Reply:
We dont need it. Removed.